lib/mcb/courses_editor.rb

```diff
@@ -71,10 +84,81 @@ def ask_route
       end
     end
 
+    def edit_qualifications
+      print_existing(:qualification)
+      update(qualification: ask_qualifications)
+    end
```

Comment:
Ya, I think that there's scope to merge these `edit_*` methods into one, or potentially with the related `ask_*`, although I understand the separation of concerns you implemented there.(

Reply:
I've now:

- generalised the `edit_*` methods in cfacaa5
- extracted a `CoursesEditorCLI` class in ca0d01a

and that seems to satisfy Code Climate.